mobile/cmp/popover/Popover.js

```diff
@@ -71,7 +72,7 @@ export const [Popover, popover] = hoistCmp.withFactory({
                                 ref: impl.contentRef,
                                 style: popper?.styles?.popper,
                                 className: 'xh-popover__content-wrapper',
-                                items: elementFromContent(content)
+                                items: elementFromContent(content, {className: popoverClassName})
```

Comment:
This is going to overwrite any className specified on the content, no?  

I suppose this is not the worst thing in the world, but the fact that it will do it even if it is undefined seems harsh. :-)

Reply:
Thanks for catching!

For some reason, I thought that `elementFromContent()` would merge classNames rather than overwrite them, like when you have a className in the HoistComponent.

I've changed it to apply the class name to the outer wrapper instead - should be much safer and doesn't modify the content in unexpected ways.